content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +93,371 @@ service:
       exporters: [datadog/api]
 ```
 
+### Environment Specific Setup
+
+#### Host:
```

Comment:
I think we should just say that they can use any of the contrib packages from the project repository latest release here and maybe have an example template. How to run the binary and the CLI interface should be on the OpenTelemetry docs.